Test/DeviceFixture.cs

```diff
@@ -0,0 +1,90 @@
+﻿/*
+This file is part of SharpPcap.
+
+SharpPcap is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+SharpPcap is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with SharpPcap.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/*
+ * Copyright 2010 Chris Morgan <chmorgan@gmail.com>
+ */
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using SharpPcap;
+using SharpPcap.LibPcap;
+using SharpPcap.Npcap;
+using SharpPcap.WinPcap;
+
+namespace Test
+{
+
+    public class DeviceFixture
+    {
+        private readonly ICaptureDevice Device;
+        public DeviceFixture(ICaptureDevice device)
+        {
+            Device = device;
+        }
+
+        public ICaptureDevice GetDevice() => Device;
+
+        public override string ToString()
+        {
+            return new string(Device.Name.Take(22).ToArray());
```

Comment:
why 22?

Reply:
I don't need the full device name, and if `ToString()` returns a very long string, Visual Studio test explorer will look weird.
with 22 chars Visual Studio Test Explorer looks like this
![image](https://user-images.githubusercontent.com/1814900/80253358-6ad91d80-8679-11ea-9bb8-7f3e767d93b5.png)

Reply:
@kayoub5 imo we shouldn't use magic numbers, could you modify the code to be something like:

var maximumDeviceNameCharactersToDisplay = 22;
blah.Take(maximumDeviceNameCharactersToDisplay);

That way we can tell from the code at least what you were intending to do.

Reply:
@kayoub5 can you make that code change here to clarify that magic number? See my above comment.

Reply:
Updated to take full `Device.Name`